src/components/Gouvernance/NoteDeContexte/NoteDeContexte.test.tsx

```diff
@@ -111,11 +112,106 @@ describe('note de contexte', () => {
+    // GIVEN
+    afficherUneGouvernanceAvecNoteDeContexte()
+
+    // WHEN
+    jOuvreLeFormulairePourModifierUneNoteDeContexte()
+
+    // THEN
+    const drawer = modifierUneNoteDeContexteDrawer()
+    const editeurDeTextEnrichi = within(drawer).getByRole('textarea')
+    expect(editeurDeTextEnrichi).toBeInTheDocument()
+    expect(editeurDeTextEnrichi.innerHTML).toBe('<p><strong>titre note de contexte</strong></p><p>un paragraphe avec du bold <strong>bold</strong></p>')
+    const boutonEnregistrer = within(modifierUneNoteDeContexteDrawer()).getByRole('button', { name: 'Enregistrer' })
+    expect(boutonEnregistrer).toBeDisabled()
+    const modifierPar = within(drawer).getByText('Modifié le 06/09/2024 par Jean Deschamps')
+    expect(modifierPar).toBeInTheDocument()
+  })
+
+  it('puis que je clique sur fermer, alors le drawer se ferme', () => {
+    // GIVEN
+    afficherUneGouvernanceAvecNoteDeContexte()
+    jOuvreLeFormulairePourModifierUneNoteDeContexte()
+    const drawer = screen.getByRole('dialog', { name: 'Ajouter un comité' })
+
+    // WHEN
+    jeFermeLeFormulairePourModifierUneNoteDeContexte()
+
+    // THEN
+    expect(drawer).not.toBeVisible()
+  })
+
+  it('puis lorsque je modifie le contenu de la note de contexte, les boutons enregistrer et supprimer deviennent actifs', () => {
+    // GIVEN
+    afficherUneGouvernanceAvecNoteDeContexte()
+    mockRichTextEditor.contenu = '<p>Ma note de contexte</p>'
+
+    // WHEN
+    jOuvreLeFormulairePourModifierUneNoteDeContexte()
+
+    // THEN
+    const boutonEnregistrer = within(modifierUneNoteDeContexteDrawer()).getByRole('button', { name: 'Enregistrer' })
+    const boutonSupprimer = within(modifierUneNoteDeContexteDrawer()).getByRole('button', { name: 'Supprimer' })
+    expect(boutonEnregistrer).not.toBeDisabled()
+    expect(boutonSupprimer).not.toBeDisabled()
+  })
+
+  it('quand je clique sur le bouton enregistrer le drawer se ferme et une notification s‘affiche', async () => {
+    // GIVEN
+    const modifierUneNoteDeContexteAction = vi.fn(async () => Promise.resolve(['OK']))
+    afficherUneGouvernanceAvecNoteDeContexte({ modifierUneNoteDeContexteAction, pathname: '/gouvernance/11' })
+    presserLeBouton('Modifier')
+    const formulaire = within(modifierUneNoteDeContexteDrawer()).getByRole('form', { name: 'labelModifierNoteDeContexteId' })
+    const boutonEnregistrer = within(formulaire).getByRole('button', { name: 'Enregistrer' })
+
+    // WHEN
+    fireEvent.submit(formulaire)
+
+    // THEN
+    expect(boutonEnregistrer).toHaveAccessibleName('Modification en cours...')
+    expect(boutonEnregistrer).toBeDisabled()
+    await waitFor(() => {
+      expect(screen.queryByRole('dialog', { name: 'Modifier note de contexte' })).not.toBeInTheDocument()
+    })
+    const notification = await screen.findByRole('alert')
```

Comment:
```suggestion
    const notification = screen.getByRole('alert')
```
Vu qu'il y a un le `waitFor` avant, tu n'as plus besoin d'await